### PR TITLE
refactor using pylint and autopep8

### PR DESCRIPTION
**Pull request by krnick**

### CODE REVIEW

File: quark/Evaluator/pyeval.py [hunk 1]
```diff
@@ -307,8 +311,10 @@ def AGET_OBJECT(self, instruction):
 
         try:
 
-            array_obj = self.table_obj.get_obj_list(int(instruction[2][1])).pop()
-            array_index = self.table_obj.get_obj_list(int(instruction[3][1])).pop()
+            array_obj = self.table_obj.get_obj_list(
```
Comment (sidra-asa) [inline quark/Evaluator/pyeval.py]:
```Python
        reg = instruction[1]
        index = int(reg[1:])
        table_obj = self.table_obj
        array_obj_ix = int(instruction[2][1])
        array_index_ix = int(instruction[3][1])
        try:

            array_obj = table_obj.get_obj_list(array_obj_ix).pop()
            array_index = table_obj.get_obj_list(array_index_ix).pop()
```

It looks better to me.

**Comment by sidra-asa** [comment]
How about request @18z for polishing in docstring ?

File: quark/Objects/ruleobject.py [hunk 2]
```diff
@@ -1,8 +1,12 @@
+# This file is part of Quark Engine - https://quark-engine.rtfd.io
+# See GPLv3 for copying permission.
+"""RuleObject is used to store the rule from json file"""
```
Comment (sidra-asa) [inline quark/Objects/ruleobject.py]: This line can be skipped.

File: quark/Objects/tableobject.py [hunk 3]
```diff
@@ -1,9 +1,15 @@
+# This file is part of Quark Engine - https://quark-engine.rtfd.io
+# See GPLv3 for copying permission.
+"""This table is used to track the usage of variables in the register"""
```
Comment (sidra-asa) [inline quark/Objects/tableobject.py]: Line 3 can be skipped

File: quark/Objects/variableobject.py [hunk 4]
```diff
@@ -1,8 +1,15 @@
+# This file is part of Quark Engine - https://quark-engine.rtfd.io
+# See GPLv3 for copying permission.
+"""The variableobject is used to record the state of each register"""
```
Comment (sidra-asa) [inline quark/Objects/variableobject.py]: Line 3 can be skipped.

File: quark/Objects/apkinfo.py [hunk 5]
```diff
@@ -22,85 +28,87 @@ def permissions(self):
...
                     bytecode_obj = None
                     reg_list = []
 
                     # count the number of the registers.
                     length_operands = len(ins.get_operands())
                     if length_operands == 0:
                         # No register, no parameter
-                        bytecode_obj = BytecodeObject(ins.get_name(), None, None)
+                        bytecode_obj = BytecodeObject(
+                            ins.get_name(), None, None)
```
Comment (sidra-asa) [inline quark/Objects/apkinfo.py]:
```Python
                    if length_operands == 0:
                        # No register, no parameter
                        bytecode_obj = BytecodeObject(ins.get_name(),
                                                      None, 
                                                      None)
```
or

```Python
                    if length_operands == 0:
                        # No register, no parameter
                        bytecode_obj = BytecodeObject(
                                           ins.get_name(),
                                           None, 
                                           None
                                       )

```

The readability is better to me.

File: tests/Evaluator/test_pyeval.py [hunk 6]
```diff
@@ -85,8 +92,10 @@ def test_invoke_virtual(self, pyeval):
             "java.io.file.close()",
             "some_function()(Lcom/google/progress/SMSHelper;,some_string)",
         ]
-        assert pyeval.table_obj.pop(4).called_by_func == v4_expected_call_by_func
-        assert pyeval.table_obj.pop(9).called_by_func == v9_expected_call_by_func
+        assert pyeval.table_obj.pop(
+            4).called_by_func == v4_expected_call_by_func
+        assert pyeval.table_obj.pop(
+            9).called_by_func == v9_expected_call_by_func
```
Comment (sidra-asa) [inline tests/Evaluator/test_pyeval.py]: Weird line breaking.

**Comment by sidra-asa** [comment]
Please check the line breaking in each modification if it is legible. 
Also, the test_main.py may be removed.

**Comment by krnick** [comment]
@sidra-asa thanks for your comment, I also feel weird some line breaking with using autopep8.
I will fix that using your comment. BTW, @18z is traveling around Taiwan now xD.

**Comment by sidra-asa** [comment]
There are remaining weird line breaking in test_pyeval.py. Please check these are appropriate.

**Comment by krnick** [comment]
thanks @sidra-asa  for reviewing 😺

**Pull request merged**